modsys/connectors/avid/cloud.py

```diff
@@ -0,0 +1,79 @@
+# Copyright: (c) 2022, Adrian Brown <adrbrownx@gmail.com>
+# Copyright: (c) 2023, ModsysML Project
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from modsys.connectors.avid.base import AbstractAVIDProvider
+
+from avidtools.datamodels.report import Report, Problemtype, LangValue
```

Comment:
These imports are redundant in the current implementation. They should ideally be used in `AVIDProvider.create_report()` to add its arg into a `Report` object then export that as json. That's no the case currently, which is fine as far as the first release is concerned. In that case, I recommend not loading these modules